src/test/java/com/thealgorithms/datastructures/graphs/WelshPowellAlgorithmTest.java

```diff
@@ -0,0 +1,48 @@
+package com.thealgorithms.datastructures.graphs;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.thealgorithms.datastructures.graphs.WelshPowell.WPGraph;
+import org.junit.jupiter.api.Test;
+
+class WelshPowellAlgorithmTest {
+
+    @Test
+    void testSimpleGraph() {
+        WPGraph graph = WelshPowell.createGraph(4);
+        WelshPowell.addEdge(graph, 0, 1);
+        WelshPowell.addEdge(graph, 1, 2);
+        WelshPowell.addEdge(graph, 2, 3);
+        int[] colors = WelshPowell.welshPowellColoring(graph);
+        assertEquals(true, isValidColoring(graph, colors));
```

Comment:
```suggestion
        assertTrue(isValidColoring(graph, colors));
```
Make sure to
```java
import static org.junit.jupiter.api.Assertions.assertTrue;
```

In general, I would suggest to make the tests more _direct_, i.e. really compare the result of `WelshPowell.welshPowellColoring` with some `int[]`.